scissors/weapon_selection.c

```diff
@@ -0,0 +1,44 @@
+/**
+ * @file       weapon_selection.c
+ * @brief      weapon_selection
+ * 
+ * @author     Leonid Chebenko
+ * @version    1.0
+ */
+
+#include "weapon_selection.h"
+#include "game_logic.h"
+#include <stdio.h>
+
+void print_weapon(int weapon)
+{
+        if (weapon == ROCK)
+                printf("ROCK\n");
+
+        else if (weapon == PAPER)
+                printf("PAPER\n");
+
+        else if (weapon == SCISSOR)
+                printf("SCISSOR\n");
+}
+
+int weapon_selection()
+{
+        int choice;
+
+        do {
+                printf("Choose your weapon\n");
+
+                printf(" [1] rock\n [2] scissors\n [3] Paper\n");
```

Comment:
Do not use too much empty lines. Add empty line to separate logical blocks in code, not after each code line (not only here).